service/tablet_allocator.cc

```diff
@@ -1497,6 +1503,12 @@ class tablet_allocator_impl : public tablet_allocator::impl
         co_return std::move(new_tablets);
     }
 
+    tablet_map set_balancing_enabled(token_metadata_ptr tm, table_id table, bool enabled) {
+        auto& tablets = tm->tablets().get_tablet_map(table);
+        tablet_map new_tablets{tablets.tablet_count()};
+        new_tablets.set_balancing_enabled(enabled);
+        return new_tablets;
```

Comment:
This function looks awkward. You return an empty tablet map, but the function is called "set". I don't think you actually use this function?

Reply:
dropped. it was a leftover of the previous try.